modules/ingestor/src/service/advisory/csaf/creator.rs

```diff
@@ -37,17 +38,22 @@ impl PackageStatusCreator {
     pub fn add_all(&mut self, csaf: &Csaf, ps: &Option<Vec<ProductIdT>>, status: &'static str) {
         for r in ps.iter().flatten() {
             for purl in resolve_purls(csaf, r) {
-                let package = Purl::from(purl.clone());
+                let mut package = Purl::from(purl.clone());
+                package.qualifiers.clear();
```

Comment:
wrt package-vs-purl, was going to wait for Jen's PR to settle (and the conversation around it).

I'm clearing the qualifiers, because we're ultimately tracking unqualified packages (purls), and the additional qualifiers break `PartialEq` by comparing against bits we shouldn't be carrying around at this point.

Basically, we have an `pkg:rpm/crap@1.2.3?arch=ppc` and another `pkg:rpm/crap@1.2.3?arch=c64` and those `?arch=...` are just noise for this process, so I strip them off, so we don't end up with `pkg:rpm/crap@1.2.3` in the queue _twice_.